apps/els_lsp/src/els_completion_provider.erl

```diff
@@ -177,9 +177,9 @@ atoms(Document, Prefix) ->
   Unique = lists:usort(Atoms),
   filter_by_prefix(Prefix, Unique, fun to_binary/1, fun item_kind_atom/1).
 
--spec item_kind_atom(binary()) -> map().
+-spec item_kind_atom(atom()) -> map().
```

Comment:
this last commit I don't fully understand
`filter_by_prefix(Prefix, Unique, fun to_binary/1, fun item_kind_atom/1).`
this converts the atom in the `id` of the POI with `to_binary` into a binary, then `item_kind_atom/1` is definitely called with a binary. It is the wrong binary though because it's missing the quotes, right?
so maybe `to_binary/1` could be fixed to call `io:write` instead? what do you think @keynslug ?

(otoh `item_kind_field` is called with a binary from `all_record_fields/2` but with an atom from `record_fields/2`, that's wring indeed, so maybe the later should be fixed to wrap it in a `to_binary` first)

Reply:
Thanks for a thoughtful review. Guess you're right, I've addressed these concerns in  dda2734 .